docs/sources/_index.md.t

```diff
@@ -54,30 +58,18 @@ hero:
 
 Collecting the relevant telemetry data, such as metrics, logs, and traces, for analysis is an indispensable part of understanding the health of your system.
 
-{{< param "PRODUCT_NAME" >}} is a multifunction tool you can use to collect, process, and forward telemetry data to the observability backend of your choosing.
+{{< param "PRODUCT_NAME" >}} is more than just a collector. You can use {{< param "PRODUCT_NAME" >}} to collect, process, and forward telemetry data to the observability backend of your choosing.
```

Comment:
Could we change this sentence to:

Alloy is software that collects, processes, and forwards telemetry data to an observability backend of your choice, where they can be stored & visualized.